regression-tests.recursor-dnssec/test_RecDnstap.py

```diff
@@ -186,7 +186,7 @@ def FrameStreamUnixListener(cls, conn, param):
                 fstrm_handle_bidir_connection(conn, lambda data: \
                 param.queue.put(data, True, timeout=2.0))
             except socket.error as e:
-                if e.errno == 9:
+                if e.errno == errno.EBADF or e.errno == errno.EPIPE:
```

Comment:
minor style suggestion - feel free to ignore:
```suggestion
                if e.errno in (errno.EBADF, errno.EPIPE):
```